modules/deploy/pages/deployment-option/self-hosted/manual/production/production-readiness.adoc

```diff
@@ -0,0 +1,534 @@
+    write_iops: 182408
+    write_bandwidth: 788050688
+```
+
+[[disk-perf]]
+=== Run self test to check disk performance
+
+Run the self test to ensure that disk performance is within an acceptable range.
+
+Example:
+```bash
+rpk cluster self-test status
+```
+
+Output:
+```bash
+NODE ID: 1 | STATUS: IDLE
+=========================
+NAME        512KB sequential r/w throughput disk test
+INFO        write run
+TYPE        disk
+TEST ID     e13b2c93-2417-458b-87be-fac409089513
+TIMEOUTS    0
+DURATION    30000ms
+IOPS        984 req/sec
+THROUGHPUT  492.1MiB/sec
+LATENCY     P50     P90     P99     P999    MAX
+            4095us  4095us  4351us  4607us  5119us
+```
+
+[[hostnames-interfaces]]
+=== Advertised hostnames use correct interfaces
+
+Ensure that the advertised hostnames are operating on the correct network interfaces.
+
+Example:
+```bash
+grep -A1 advertised /etc/redpanda/redpanda.yaml
+    advertised_kafka_api:
+    -   address: 54.244.110.128
+--
+    advertised_rpc_api:
+        address: 172.31.40.237
+```
+
+[[continuous-db]]
+=== Continuous data balancing is configured
+
+Ensure that xref:manage:cluster-maintenance/continuous-data-balancing.adoc[Continuous Data Balancing] is configured and enabled.
+
+Example: 
+```bash
+rpk cluster config get partition_autobalancing_mode
+```
+
+Output:
+```bash
+continuous
+```
+
+[[debug-bundle]]
+=== Generate a debug bundle
+
+Ensure that you’re able to generate a debug bundle from each host and upload to Redpanda Support.
+
+```bash
+sudo rpk debug bundle
+Creating bundle file...
+Debug bundle saved to '1711372017-bundle.zip'
+```
+
+[[topic-rf]]
+=== Topic replication factor
+
+Ensure that the topics replication factor are greater than one.
+
+Example:
+```bash
+rpk topic list
+```
+
+Output:
+```bash
+NAME   PARTITIONS  REPLICAS
+bad    1           1
+good   1           3
+```
+
+[[maintenance-mode]]
+=== Brokers in maintenance mode
+
+Ensure that no brokers are in maintenance mode.
+
+Example:
+
+```bash
+rpk cluster maintenance status
+```
+
+Output:
+
+```bash
+NODE-ID  ENABLED  FINISHED  ERRORS  PARTITIONS  ELIGIBLE  TRANSFERRING  FAILED
+1        false    -         -       -           -         -             -
+2        false    -         -       -           -         -             -
+3        false    -         -       -           -         -             -
+```
+
+[[decom-state]]
+== Brokers in a decommissioned state
+
+Ensure that no brokers are in a decommissioned state.
+
+Example:
+```bash 
+rpk redpanda admin brokers list
+```
+
+Output:
+```bash
+NODE-ID  NUM-CORES  MEMBERSHIP-STATUS  IS-ALIVE  BROKER-VERSION
+0        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+1        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+2        1          active             true      v23.2.25 - 8530f7c17346eb7c7459c3912f012299f3762efd
+```
+
+
+== Level 2 Production Readiness
```

Comment:
```suggestion
== Level 2 production readiness
```

Reply:
It's a bit incongruous for the checks here to not appear as subheadings as they do in the Level 1 section. We should be consistent wrt format. Also, every check should include links to help users who might be unsure about how to configure or use the items we list.

Reply:
@Deflaimun Please make each check in Level 2 readiness a subhead just like we have in Level 1. 
@tmgstevens @hcoyote Seems like we should clearly distinguish how this level differs from Level 1 and Level 3 (perhaps at the beginning of the topic where we introduce all the levels). Thoughts?